src/client/datascience/interactive-ipynb/trustCommandHandler.ts

```diff
@@ -41,6 +41,13 @@ export class TrustCommandHandler implements IExtensionSingleActivationService {
         const context = new ContextKey('jupyter.trustfeatureenabled', this.commandManager);
         context.set(true).ignoreErrors();
         this.disposables.push(this.commandManager.registerCommand(Commands.TrustNotebook, this.onTrustNotebook, this));
+        this.disposables.push(
```

Comment:
I think we should get this reviewed by a PM.
Generally I ton'd think its a good idea to display such messages. One could argue if the title indicates its trusted, then wy allow clickcing the button just to always (100% of time) display a message.
If the message is required, then i'd argue the meaning of the icon is not obvious to users.

@claudiaregio /cc

Reply:
@DonJayamanne David talked to Claudia per his comment above.

Reply:
> Claudia's ruling: have the notification pop on saying its already trusted.
"This notebook is already trusted."

Reply:
Claudia just pinged me & stated that we need an icon that cannot be clicked.
I.e. just disable it and display tooltip.

She was under the impression we cannot disable it